generators/resource/templates/actions/resource-use_model.go.tmpl

```diff
@@ -0,0 +1,166 @@
+package actions
+
+import (
+	"{{.importPath}}/models"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/markbates/pop"
+)
+
+// This file is generated by Buffalo. It offers a basic structure for 
+// adding, editing and deleting a page. If your model is more 
+// complex or you need more than the basic implementation you need to
+//  edit this file.
+
+// {{.camel}}Resource is the resource for the {{.singular}} model
+type {{.camel}}Resource struct{
+	buffalo.Resource
+}
+
+// List gets all {{.camel}}. This function is mapped to the the path 
+// GET /{{.under}} 
+func (v {{.camel}}Resource) List(c buffalo.Context) error {
+	// Get the DB connection from the context
+	tx := c.Value("tx").(*pop.Connection)
+	{{.downFirstCap}} := &models.{{.camel}}{}
```

Comment:
This should be plural.

It's generating `user := &models.User{}`, but it should be `users := &models.Users{}`